src/views/PDFView.vue

```diff
@@ -48,17 +48,22 @@ export default {
 		},
 
 		isDownloadable() {
-			if (!this.file.shareAttributes) {
+			if (!this.file || !this.file.shareAttributes) {
 				return true
 			}
-
-			const shareAttributes = JSON.parse(this.file.shareAttributes)
-			const downloadPermissions = shareAttributes.find(({ scope, key }) => scope === 'permissions' && key === 'download')
-			if (downloadPermissions) {
-				return downloadPermissions.value
+			try {
+				const shareAttributes = JSON.parse(this.file.shareAttributes)
+				const downloadPermissions = shareAttributes.find(({ scope, key }) => scope === 'permissions' && key === 'download')
+				if (downloadPermissions) {
+					return downloadPermissions.value
+				}
+				return true
+			} catch (e) {
```

Comment:
The `try { } catch` should enclose only the `const shareAttributes = JSON.parse(this.file.shareAttributes)` statement, as it is guarding against a parsing error, and the rest of statements should be safe to execute (at least if the function returns early in the `catch` like it does now).